.github/CONTRIBUTING.md

```diff
@@ -75,75 +80,47 @@ poetry install
 
 Make sure you have the Python code formatter `black` installed as it is used in a pre-commit hook. Run `pip install black` if needed.
 
-## Start the UI
+## Start the Chainlit server from source
+
+You need to build the UI once before starting the server.
 
 ```sh
-pnpm run buildUi
-cd src/chainlit/frontend
-pnpm run dev -- --port 5174
+npm run buildUi
```

Comment:
pnpm

Reply:
I think `pnpm` is required for installation. We can still use `npm` other commands. Typically I tried to run `pnpm run dev -- --port 5174` but it was not accepting the port argument.

Reply:
Yes, you should remove the extra `--`. My bad, did not see this when I replaced npm.
![Capture d’écran 2023-07-24 à 13 52 20](https://github.com/Chainlit/chainlit/assets/18298759/2dcf034b-a191-4af6-8ca7-b6df27f2293a)